data_structures/binary_tree/binary_tree_mirror.py

```diff
@@ -0,0 +1,42 @@
+# Created by susmith98
+
+
+"""
+Problem Description:
+Given a binary tree, return it's mirror.
+"""
+
+
+def binary_tree_mirror_dict(root: int, binary_tree_mirror_dictionary: dict):
```

Comment:
doctests?

Kinda counterintuitive that the params of this function are root and then the tree and the next function, the order is reversed.  Can both functions share a parameter order?

Reply:
```binary_tree_mirror``` is the first caller function as it has some validations to do, doctests are added to it. 
Yes will changes the parameters order